test/verify/check-login

```diff
@@ -60,6 +60,19 @@ account    required     pam_succeed_if.so user ingroup %s""" % m.get_admin_group
             b.set_checked('#authorized-input', True)
             b.click('#login-button')
 
+        # Test banner
+        if m.image != "rhel-8-1-distropkg": # Added in #13237
+            # Test that we don't show banner when not specified
+            b.wait_visible("#login-user-input")
+            b.wait_not_visible("#etc-issue")
+
+            m.execute("printf '[Session]\nBanner = /etc/issue\n' >> /etc/cockpit/cockpit.conf")
+            m.restart_cockpit()
+            b.reload()
+            b.wait_visible("#login-user-input")
+            b.wait_visible("#etc-issue")
+            self.assertEqual(b.text("#etc-issue pre"), m.execute("cat /etc/issue"))
```

Comment:
Hmm, I would have expected that we trim whitespace from that file, to not get ugly leading or trailing empty lines.

Reply:
I did trimming from end. There may be some reason why someone wants some whitespaces at beginning (like the issue we had that we broke some ascii art on motd).

Reply:
Exactly, thanks!